lightningd/peer_control.c

```diff
@@ -1202,8 +1202,8 @@ void peer_connected(struct lightningd *ld, const u8 *msg)
 
 	/* Log and update remote_addr for Nat/IP discovery. */
 	if (hook_payload->remote_addr) {
-		log_peer_info(ld->log, &id, "Peer says it sees our address as: %s",
-			      fmt_wireaddr(tmpctx, hook_payload->remote_addr));
+		log_peer_debug(ld->log, &id, "Peer says it sees our address as: %s",
+			       fmt_wireaddr(tmpctx, hook_payload->remote_addr));
```

Comment:
extraneous space?

otherwise ACK a5b9830796b4fe94feb55d2a05c51e5fd156dd32

I had been using exactly this on v0.11.2 because #5435 was annoying me. I didn't submit this because I thought people would think this was too blunt an instrument but I'm happy if people are OK with this.

Reply:
The space is actually 'correct' :)